.gitignore

```diff
@@ -41,3 +41,5 @@ db
 
 #goodparts symlink
 .eslintrc.js
+
+compiled.js
```

Comment:
for "convention" (see: http://browserify.org/ ) would it be an _idea_ to _name_ the compiled file `bundle.js` ? 💡